assets/js/blocks/product-search/block.js

```diff
@@ -170,9 +170,9 @@ ProductSearchBlock.propTypes = {
 	 */
 	instanceId: PropTypes.number,
 	/**
-	 * Whether this is the block preview or frontend display.
+	 * Whether it's in the editor or frontend display.
 	 */
-	isPreview: PropTypes.bool,
+	isEditor: PropTypes.bool,
```

Comment:
Right, all components should have propTypes definitions, but we have been quite inconsistent with that in the past. I [added definitions](117490b90e86b8219b3b01436c8e8f94833dfb40) to the components modified in this PR, but it might be worth creating an issue similar to #1280, listing all components that are missing propTypes.